scripts/compose.py

```diff
@@ -2012,7 +2057,7 @@ def _content(self):
                 "ELASTIC_APM_SERVICE_NAME={}".format(self.service_name),
                 "DATABASE_URL=postgres://postgres:verysecure@postgres/opbeans-ruby",
                 "REDIS_URL=redis://redis:6379",
-                "ELASTICSEARCH_URL=http://elasticsearch:9200",
+                "ELASTICSEARCH_URL={}".format(self.es_urls),
```

Comment:
like the logstash case, needs to be joined or an entry picked

Reply:
https://github.com/elastic/apm-integration-testing/pull/557#discussion_r306676939